src/kuka_control/scripts/camera.py

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/python
+# Copyright (c) 2015, Rethink Robotics, Inc.
+
+# Using this CvBridge Tutorial for converting
+# ROS images to OpenCV2 images
+# http://wiki.ros.org/cv_bridge/Tutorials/ConvertingBetweenROSImagesAndOpenCVImagesPython
+
+# Using this OpenCV2 tutorial for saving Images:
+# http://opencv-python-tutroals.readthedocs.org/en/latest/py_tutorials/py_gui/py_image_display/py_image_display.html
+
+# rospy for the subscriber
+import rospy
+# ROS Image message
+from sensor_msgs.msg import Image
+# ROS Image message -> OpenCV2 image converter
+from cv_bridge import CvBridge, CvBridgeError
+# OpenCV2 for saving an image
+import cv2
+import time
+
+# Instantiate CvBridge
+bridge = CvBridge()
+
+t0 =int(time.time())
+
+def image_callback(msg):
+    print("Received an image!")
+    try:
+        # Convert your ROS Image message to OpenCV2
+        cv2_img = bridge.imgmsg_to_cv2(msg, "bgr8")
+    except CvBridgeError, e:
+        print(e)
+    else:
+        # Save your OpenCV2 image as a jpeg 
+        # if t < 0:
+        t = str(int(time.time())-t0+1)
+        #print(t)
+        # else:
+        if int(t) <= 30:
+            cv2.imwrite('/home/muhammad/kuka_robot_urdf/src/rrbot_control/pictures/camera_image'+t+'.jpeg', cv2_img)
```

Comment:
hardcode that will not work on my laptop

Reply:
I will try to solve this problem